packages/samples/headless-ssr/app/common/engine.ts

```diff
@@ -0,0 +1,16 @@
+import {getSampleSearchEngineConfiguration} from '@coveo/headless';
```

Comment:
These files should probably not be nested under `app`, since that signifies that they are pages.

You should move `common` and `components` to the root of the project. If you want, you can add a `src` directory which contains `app`, `common` and `components`.